test-requirements.txt

```diff
@@ -0,0 +1,2 @@
+odoo14-addon-stock_location_zone @ git+https://github.com/phuctranerp/stock-logistics-warehouse.git@14.0-mig-stock_location_zone#subdirectory=setup/stock_location_zone
```

Comment:
please remove these once #1077  is merged. Also, next time, please link the depending PRs in the description ;)